### DISCUSSION TITLE
《916篇性侵儿童判决书的背后：隐秘的角落，谁来保护孩子们？》

### DISCUSSION THREAD

**Discussion by wanfanghou** [data-journalism/data-journalism.github.io#55] [Show and tell]
> **也许在未来的某一天，我们每个人都可以为隐秘的角落点起一盏灯。**

## 作品链接

[https://598346469.wixsite.com/protectchildren](https://598346469.wixsite.com/protectchildren)

![image](https://user-images.githubusercontent.com/63247802/138773747-87a960f4-942a-4b8c-92b1-103c1a022840.png)

## 作品简介

- **作品名称**：《916篇性侵儿童判决书的背后：隐秘的角落，谁来保护孩子们？》/《Hidden Corner, Who Can Protect the Children?》

- **作品内容**：作者通过分析近年来的性侵儿童的判决书，并重点分析了2019年的916篇文书，去揭开儿童性侵案背后的各种细节，最终给出我们能做的建议。

- **数据来源**

  - 中华社会救助基金会儿童安全基金女童保护项目

  - 北大法宝

  - 《法制日报》

## 创作缘由

2013年5月的时候，海南万宁某小学被爆出性侵事件，犯罪嫌疑人正是小学的校长和一位政府职员，受害者是6名小学六年级的女孩。

这个事件可以说是一个标志事件，因为当时引起轩然大波，让全社会都关注到“性侵儿童”这一事件。

随后，媒体又爆料出更多的类似案件；大众也终于认识到，“性侵儿童”这种犯罪现象，可能就发生在我们的身边。

今天是2021年，离2013年的广宁事件已经过去整整7年，在这7年里，性侵儿童这一话题一次又一次登上舆论热点，又一次又一次被时间冲洗和淡忘。

我们想要知道，近些年，这些性侵儿童事件，离我们的生活有多远？那些性侵儿童的恶魔们，到底是怎样的面孔？那些无辜的孩子，在我们不知晓的隐秘角落，遭遇了什么样的痛苦？

以及更重要的问题是，在愤怒之后，我们已经做了什么来保护这些孩子们？我们还可以做什么来保护孩子们？

基于这样的目标和背景，作者搜集了近年来关于**性侵儿童的判决书**，并且**重点分析了2019年的916篇文书**，试图解答这些问题。

### （1）没有被聚光灯照亮的角落

首先，我们从数量上看，媒体曝光的案例仅仅是实际案例的极小部分。

自2013年起，中华社会救助基金会儿童安全基金女童保护项目，对每年媒体公开报道的性侵儿童事件进行统计分析。仅仅2019年，被曝光的案件就有**301**起，然而这只是冰山一角。根据北大法宝可查的法律文书以及《法制日报》的报道，可以发现2019年法院实际审结案件数量为**4159**起，在“北大法宝”上有案件条目可查的数量为2942件，公开文书的为920件，**被曝光的案例仅占实际案件数量的7.23%**。

有趣的是，为了严谨，作者统计了四类性侵儿童的**案件数量**，分别是：**媒体报道的案件数量**、**有文书可查的案件数量**、**被登记在网上的案件数量**以及**法院审结的案件数量**。

而且作者采用“总分”的结构，一张总图，四张子图。

优点是

1. 既可以比较四类案件数量的差异与趋势

2. 也可以分别查看具体的每类案件数量的演变情况。因为如果全放一张图，数据量纲不太一致，会难以识别具体的直线走向，分成四个图来展示会更清晰一些。

缺点是数据之前存在重叠交叉。

#### 1.1 各年案件数量总图

![image](https://user-images.githubusercontent.com/63247802/138773867-d68984d0-e7e4-4a9f-aab8-41edcb528acf.png)

从图上可见

1. 从总体趋势上看，性侵儿童的案件数量呈逐年上升趋势；一方面可能是犯罪事件本身在增多；另一方面也可能是，随着社会进步和完善，越来越多的事件被披露和勇敢的说出。

2. 被媒体报道的案例长期远远落后于实际发生的案例数量；一方面，媒体出于自身利益考虑，会选择性报道典型案例；另一方面，媒体对于性侵儿童案件的关注度本身不够高。

#### 1.2 媒体有报道的案件数量

![image](https://user-images.githubusercontent.com/63247802/138773899-6824f173-f290-4b23-953b-f93129db8868.png)

从媒体报道的数量来看，2013年广宁事件后，儿童性侵的报道案例数量直线上升，且长期保持在较高位置。

这在某种程度上是一个好的现象，至少媒体提高了对儿童性侵的报道，有越来越多的人可以了解到这些案件的故事。

#### 1.3 有文书可查的案件数量

![image](https://user-images.githubusercontent.com/63247802/138773934-fdab0419-4d87-4b5d-96b0-5b2fd9fd80f8.png)

随着案件数量的增多，有文书可查的案件数量也在增多，我们可以通过详细文书查看到明确的案件细节。

#### 1.4 被登记在网络上的案件数量

![image](https://user-images.githubusercontent.com/63247802/138773971-4948b97f-d3c0-4367-9492-c01567325044.png)

被登记在网络上的案件数量也在增多。

#### 1.5 法院审结数量

![image](https://user-images.githubusercontent.com/63247802/138773998-03a71121-beee-445e-bf4d-3b21b9e46d28.png)

法院审结的案例数量也在增多。

- 从数量上看，性侵儿童案例数量在近年不断攀升。

- 但我们需要注意的是，受到社会环境、新闻传播观念等因素的影响，性侵儿童事件不一定会被披露，而且有可能因为当事人或周围人的消极情绪或认知水平的受限而不为人知。

- 随着年龄的增长、思想观念的变化，这类案件越来越多的进入我们的视野。

#### 1.6 全国性侵儿童案件数量分布

从性侵案件发生的地理位置上看，1998年至2019年期间，浙江省累计审理758起，四川省累计审理457起，上海市累计审理347起，为全国前三，而相对来说，西藏自治区累计审理2起，海南省累计审理15起，青海省累计审理19起，则数量较少。

![image](https://user-images.githubusercontent.com/63247802/138774035-3cf51103-53ea-4434-a602-1998f87dcb1a.png)

- 缺点：自动播放时间，不能手动调整

#### 1.7 学术关注度&媒体关注度

从性侵案件的学术关注度和媒体关注度来看，我们可喜的发现，自2013年，在广宁事件的影响下，关于儿童性侵的文献数量与报道数量激增。性侵儿童案件的媒体报道进入了包容多元期，大量的社论、报纸评论、学术文献涌出，大众认知度提高。

![image](https://user-images.githubusercontent.com/63247802/138774057-6ad855dd-ffb7-460f-ac05-b88e6cb0f3dd.png)

### （2）性侵案件案件当事人都是谁？

![image](https://user-images.githubusercontent.com/63247802/138774085-e1864bae-da76-4fd7-a74c-af4d31311ff4.png)

了解了儿童性侵逐渐被关注的现状后，我们更想要知晓的，是这些性侵案件中的当事人是怎样的。

数据显示，大部分性侵施害者，和被害者都**比较熟悉**，只有**10%为陌生人**，约有**30%为儿童的亲戚**，例如叔叔伯伯；**60%为其他熟人**，如父母的朋友，保姆或邻居；

#### 2.1 当事人关系图

![image](https://user-images.githubusercontent.com/63247802/138774115-ef1dc637-ae38-4d58-a7fd-fdebde22c077.png)

在我们搜集到的213条有施害者和受害者关系的记录的文书中，有74条记录为师生关系，占比34%，53条记录为亲戚关系，占比24%，其中18条记录为继父女（或母亲同居男友），6条为亲生父亲。

家庭内部性侵事件并非个例，这53条记录更有可能只是此类事情的冰山一角。我国“以家庭为中心”的价值观可能会掩盖家庭内部的儿童性侵犯案件，也加大了我国司法部门干预这一问题的难度。家庭内部的性侵更隐秘、更难被发现，当亲密的互动发生在家庭成员之间，复杂的关系和情感不仅会使性侵的边界变得模糊，也会使披露或举报更加困难。

![image](https://user-images.githubusercontent.com/63247802/138774163-6f32961e-2f38-47a3-a866-4893f3da2847.png)

  男性并不是不会被性侵。2019年，BBC纪录片《被性侵的男性：打破沉默》披露了一组令人触目惊心的数据：每小时，约有8名男性遭到性侵。在人生的不同阶段，1/6的男性遭到不同程度的性侵。

  #### 2.2 施害者曾犯罪情况

![image](https://user-images.githubusercontent.com/63247802/138774194-73dfefa2-c537-46f5-8d79-0254e7b4d1cd.png)

数据显示，8.5%的施害者并非首次犯罪，在他们曾经犯过的罪中，占比最高的为盗窃罪，出现23次，其次为猥亵儿童/奸淫幼女罪和强奸罪，分别出现22次和17次。

#### 2.3 性别分布

虽然从数据上显示，男性儿童遭遇性侵事件的数量远低于女性儿童，但男生并不是不会被性侵。

![image](https://user-images.githubusercontent.com/63247802/138774244-a98c89d5-5b32-401c-affb-686536b2ddfe.png)

2019年，BBC纪录片《被性侵的男性：打破沉默》披露了一组令人触目惊心的数据：**每小时，约有8名男性遭到性侵**。在人生的不同阶段，**1/6的男性**遭到不同程度的性侵。

可是**90%的受害者**对此穷其一生保持缄默——对于大多数男性性侵受害者而言，“女性是受害者、男性是加害者”的固有二元对立假设，阻碍了许多男性求助，**“性别”**是他们创伤中至关重要的一环，也是他们难以开口的痛点。

值得注意的是，在50起受害者为男性的案例中，有9起是“网站预约男童有偿性服务”。施害者通过网络，或是通过男童性服务中介介绍，与男童取得联系，在旅馆等约定地点对男童进行性侵。由于我国法律界定的强奸罪定义为“以暴力、胁迫或者其他手段强奸妇女”，因此当受害者为男性时，尽管发生了事实上的奸淫，但无法被判定为强奸罪。

同时我们发现，在这9起“网站预约男童有偿性服务”案件中，有6起案件的受害者都签署了谅解书，获得20000元至60000元不等的赔偿金额，达到了2/3，而对于整体916起案件来说，因赔偿而谅解的案件比例为19.76%。目前在我国，“娈童恋”还未引起足够的重视，通过网络进行的男童有偿性服务需要被关注。

### （3）角落里，到底发生了什么？

![image](https://user-images.githubusercontent.com/63247802/138774270-66245a97-23fc-4196-8995-fb01a440d16d.png)

#### 3.1 关键词分布

从判决书的关键词分布来看，生殖器、胸部、嘴部、臀部，是被提及次数最多的遭到猥亵的部位。猥亵方式中，触摸、亲吻、搂抱、生殖器插入、抠摸、生殖器摩擦出现频率较高，同时，一些没有直接触碰被害人的猥亵方式也应该被注意，例如给儿童观看色情录像、拍摄儿童的裸体照片或视频、言语猥亵。

![image](https://user-images.githubusercontent.com/63247802/138774291-3dd3203c-5a80-4a19-9f55-e9e73ab0f455.png)

利用**互联网性侵未成年人**的案件也亟需关注。犯罪嫌疑人通过网络聊天等软件结识未成年人，利用未成年人认知、辨别和反抗能力较差的特点，采取欺骗、诱惑等方式获得未成年人的裸照或者视频，甚至进一步发展到在现实中的性关系。

然而与之相对的是，家长对于儿童使用手机、电脑等上网的监管存在缺位。在2018年及2019年的“女童保护”报告中，都只有大约**42%**的家长会对孩子上网进行严格监管。

![image](https://user-images.githubusercontent.com/63247802/138774315-42940975-05cb-40c7-a602-934121e8e781.png)

#### 3.2 被猥亵人数与次数

**916**件案例中，我们一共统计到**1349**名受害人（注：“多人”算为5人，“十余人”算为15人），平均每起案件有1.4名，而最多的一起案件有21名受害人。平均受到猥亵次数为2.24次（注：多次算5次，十余次算15次），最多的一起有70次。

![image](https://user-images.githubusercontent.com/63247802/138774336-bffa7077-926e-4917-b087-888a560822c1.png)

#### 3.3 猥亵地点

性侵儿童案的特点之一就是隐蔽性强，在916份判决书中，猥亵最常发生的地点是家中，其次是学校，以及车内、楼道、公厕，被判定为在公共场所发生的案件数量为58起，占全部案件数量的6.3%。

![image](https://user-images.githubusercontent.com/63247802/138774349-733993ff-77f4-4c36-adce-68873ffc25d6.png)

- 可视化的方式展现地理位置

### （4）噩梦之后，会有正义到来吗？

![image](https://user-images.githubusercontent.com/63247802/138774369-c4179a69-501d-48a1-8972-944d1614d3ce.png)

> 上市公司新城控股董事长王振华，因涉嫌性侵9岁女童，被判犯猥亵儿童罪，有期徒刑5年，判决结果一出便引发舆论风暴，民意一边倒地认为量刑偏轻。凤凰网“王振华猥亵女童被判5年你怎么看”的调查结果显示，超7成网民认为量刑太轻，另有近2成的网民认为王振华应当在狱中度过余生。

我们经常想问：该做什么来阻止这样的悲剧一次次重演？当挥起法律之利剑，我们是庆幸“持正义之天平”，还是哀叹“多少罪恶假汝之名”？

王振华案的判刑是否得当？其他的性侵儿童案件，都被判处了什么样的刑罚？

#### 4.1 时间轴

![image](https://user-images.githubusercontent.com/63247802/138774392-3254573c-5458-476c-8135-7325003b16cf.png)

从“北大法宝”的数据看，性侵儿童案大部分都是按猥亵儿童罪定罪处罚，也有与强奸罪、强制猥亵罪等罪名数罪并罚的情况。13.6%的案件都涉及数罪并罚，在这其中73.6%为强奸罪，也就是说，916个案件中，有10%的儿童都受到了法律概念上的强奸，即双方生殖器发生接触。

单从猥亵儿童罪的量刑来看，刑期多分布在1年-4年，案件的平均刑期为28.84个月，也就是两年四个月。当被判为强奸罪时，平均刑期为73.74个月，也就是六年两个月左右。

#### 4.2 刑期分布

![image](https://user-images.githubusercontent.com/63247802/138774416-de5c399c-f5a5-4964-977d-e8eb68fef032.png)

#### 4.3 犯罪人职业限制

另一个值得注意的处罚是对于施害者的从业限制。916起案件中有21起的刑罚中提及了“禁止从事未成年人培训工作/教育工作/服务工作/相关工作“，禁止年限从3年到5年不等。三年至五年的禁业规定，对于预防职业犯罪再犯具有积极意义，但对于预防性侵未成年人犯罪而言，性侵未成年人罪犯在刑满释放五年之后，依然可以不受限制地进入学校以外的教育培训机构，依然可以有与未成年人密切接触的机会，其性侵未成年人犯罪再次发生的可能性依然存在。
美国司法部公开的统计数据显示，17%有性侵未成年人前科的人出狱后还会再次犯罪，如果这些人接触未成年人机会比较大，这一比例还会更高。
在我国刑法酌定量刑中，受害者签署刑事谅解书可以使施害者获得一定程度上的从轻处罚。

![image](https://user-images.githubusercontent.com/63247802/138774447-4f4715da-6988-4dc1-bfe6-2f64fbef1c6d.png)

我们发现在916篇文书中，共有181个案件的受害人出具谅解书，占总数的19.76%，谅解原因多为施害者及其家属对受害者进行了赔偿。

但同时，令人心痛的是，另一个常见的谅解原因是被害人与施害人有着紧密的家庭关系，被害人不得不签署谅解书。例如在《杨吉高强奸罪一审刑事判决书》中，被告人杨吉高多次性侵继女，长达两年，但最终受害人因家庭关系的压力而出具了谅解书。

### （5）如果光会照亮这里

![image](https://user-images.githubusercontent.com/63247802/138774472-62b95b4a-e697-4971-a297-b8dcee0c0eb5.png)

在《2014年儿童防性侵教育及性侵儿童案件统计报告》中，家长没有对孩子进行性教育的原因有“孩子太小了”（85.8%）、“想教育但不知如何开口”（8.6%）、“怕教坏孩子”（2.5%）。我们的社会一直以来对“性”避之若浼，然而就像网上流传的那句话“你嫌性教育太早，坏人不会嫌你孩子太小”一样，某些程度上来说，性教育的匮乏成为了助长性犯罪的帮手。

#### 5.1 家长不对孩子做性教育的原因

![image](https://user-images.githubusercontent.com/63247802/138774500-cb0f54da-507c-4c55-a8ea-acd518611696.png)

好在令人欣慰的是，综合分析2014-2019年“女童保护”的调查报告，我们发现家长逐渐产生了对孩子进行性教育的意识，对孩子进行性教育的家长逐年增多。在2014年，有51.4%的家长没有对孩子进行过性教育，而在2019年，这一比例减少到了22.24%。

#### 5.2 性教育普及率

![image](https://user-images.githubusercontent.com/63247802/138774517-09c41ab4-81b8-4dcb-b07f-c0a9a7da6acd.png)

不仅仅是性教育需要被普及，在人们面对性侵受害者时的观念和态度也应转变。某性侵受害者说：“十六岁那年被强奸之后，我向家人求助。他们是这样’开导’我的：’你不要总想着自己是个受害者，你就当是个参与者好了。'这句话让我感到自己又被强奸了一次。回想起来，这句话甚至比被强奸本身还要痛。后来我才明白，这叫做’二次伤害’。”
家人、朋友、法律机构、大众舆论都有可能对受害者造成二次伤害，这样的伤害甚至会比性侵本身更严重。

#### 5.3 关键词抽取

![image](https://user-images.githubusercontent.com/63247802/138774529-640b68e6-6b3e-4951-8235-e5b4a41c2443.png)

![image](https://user-images.githubusercontent.com/63247802/138774561-1635935a-768e-4563-859b-1d99d7ea786c.png)

我们首先需要保持对受害者的同理和共情，再痛斥性犯罪本身。

作为普罗大众中的一员，我们希望性侵儿童案件不再是一件需要被掩埋在受害者羞耻、痛苦的目光里的事情，不再是一件会被舆论评价为受害者“一生的污点”的事情，不再是一件猎奇、戏谑、可以开黄色玩笑的事情。

我们希望尽我所能地做到我们能为孩子们做的事情，剩下的，交给时间，交给法律。

也许在未来的某一天，我们每个人都可以为隐秘的角落点起一盏灯。

## 可视化效果分析

1. 作品整体色调为深绿，渲染压抑低沉的基调

2. 在作品中穿插了很多图片以及卡通人物形象，丰富了可视化的效果

3. 在各年案件数量部分，采用“总分”的结构，分别展示四类具体的案件数量，兼顾总体变化以及个体趋势。

4. 在全国性侵儿童案件数量分布图中，地图的时间设置为自动默认，较为死板，不便于查看特定年份或实现年份对比。

5. 在当事人关系图中，采用两层嵌套式的饼状图，便于突出数据的层级结构，清晰准确。

6. 关键词云的设计具有人文关怀，抨击社会对儿童的侵害。

7. 儿童性侵的高发地点图，采用可视化的卡通形象，生动形象展现。

## 技术实现

1. Flourish：[https://flourish.studio/](https://flourish.studio/)

2. 网站搭建：[https://www.wix.com/lpviral/enviral?utm_campaign=vir_wixad_live&adsVersion=white&orig_msid=a3f7c108-6a63-4e05-a6e2-a286757bee96](https://www.wix.com/lpviral/enviral?utm_campaign=vir_wixad_live&adsVersion=white&orig_msid=a3f7c108-6a63-4e05-a6e2-a286757bee96)

## 总结：为什么选择这个话题

1. 从展现形式来看，很引人入胜，打动人

2. 从数据上来看，不是简单的罗列数据和机械的做图表，而是用心的了解和分析数据背后的故事。这个点非常重要。

> 熊丙奇：其实这些案件一直存在，只是新闻工作者没有意识去报道，又因为过去媒体报道的渠道相对单一，导致很多类似案件虽然发生，却未能进入公众视野

3. 希望能够让更多的人关注到这个事情

4. 曝光可能也是一种伤害？

**Comment by chengjun** [comment]
2017 The Al Neuharth Innovation in Investigative Journalism Award, Large Newsroom FINALIST

# Doctors & Sex Abuse

https://awards.journalists.org/entries/doctors-sex-abuse/

**Reply by chengjun** [reply]
https://datajournalism.com/read/handbook/two/working-with-data/making-algorithms-work-for-reporting Making Algorithms Work for Reporting
Written by Jonathan Stray
